katportalclient/client.py

```diff
@@ -339,7 +458,7 @@ def _cache_jsonrpc_request(self, jsonrpc_request):
                     requests_to_remove.append(req)
         elif (jsonrpc_request.method.startswith('set_sampling_strat') and
               jsonrpc_request.params[2] == 'none'):
-            # index 2 of params is always the sampling strategy
+            # index 1 of params is always the sampling strategy
```

Comment:
I was mistaken when I started testing this and never reverted the comment. Should be index 2. Changed inad99d45